src/styles/typography.ts

```diff
@@ -1,10 +1,19 @@
 import { StyleSheet } from "aphrodite";
-import "assets/fonts/fonts.css";
+
+export const fonts = {
+  roboto: "Roboto",
+  ubuntu: "Ubuntu",
+};
 
 export const typography = StyleSheet.create({
   body1: {
     fontSize: 12,
     lineHeight: "16px",
-    fontFamily: "Roboto",
+    fontFamily: fonts.roboto,
+  },
+  error1: {
```

Comment:
try to keep it differentable like every `body1` will have roboto, and then u should create `altBody1`, which has only one property different -> fontFamily

Reply:
try to avoid creating one-time objects like error1 if you can just create the altBody1